oauth2_http/java/com/google/auth/oauth2/UserAuthorizer.java

```diff
@@ -205,8 +206,8 @@ public UserCredentials getCredentials(String userId) throws IOException {
     Long expirationMillis =
         OAuth2Utils.validateLong(tokenJson, "expiration_time_millis", TOKEN_STORE_ERROR);
     Date expirationTime = new Date(expirationMillis);
-    String scopes =
-        OAuth2Utils.validateOptionalString(
+    List<String> scopes =
+        OAuth2Utils.validateOptionalListString(
```

Comment:
For the existing token with string-typed scopes in the token store, this validation will fail, and they need to refresh to get a new access token?

Reply:
The feature is not shipped yet, it was checked in late last year. So we don't change the behavior yet.

Reply:
added this detail to the change description

Reply:
This makes sense as we do not need to worry about breaking existing customers.